tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -131,6 +131,11 @@ trait VirtualSerializedScalaJSIRFile extends VirtualBinaryFile with VirtualScala
     try {
       ir.InfoSerializers.deserialize(stream)
     } catch {
+      case e: ir.IRVersionNotSupportedException =>
+        throw new IOException(
+          s"Failed to deserialize info of file compiled with Scala.js ${e.version}" +
+          s" (supported: ${e.supported.mkString(", ")}): $path", e)
```

Comment:
4 spaces.

Also, why not rethrow an `IRVersionNotSupportedException` (with a custom message)?